### PR TITLE
add firefox-health-dashboard

### PR DESCRIPTION
**Pull request by klahnakoski**
first attempt at adding project

### CODE REVIEW

File: src/data/firefox-health-dashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Firefox Health Dashboard
+incon: https://codetribute.mozilla.org/projects/reporting
```
Comment (helfi92) [inline src/data/firefox-health-dashboard.yaml]: What is `incon`?

File: src/data/firefox-health-dashboard.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  The [main Github page](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/#developing)
+  has instructions on how to begin. You will require `node` and `yarn` installed.
+
+  ### How Do I Write the Code?
+
+  1. [Find an Issue to work on](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues) or
+  [looks at the site](https://health.graphics/) and see if there is something small you would like to fix.
+  2. Fork-and-clone the repo, make some change, and submit a PR. Someone will be happy to guide you
+  through the process of coding and getting it acepted.
```
Comment (helfi92) [inline src/data/firefox-health-dashboard.yaml]: `s/acepted/accepted/`.

File: src/data/firefox-health-dashboard.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  ## How Do I Get Help?
+
+  Anyone on `#frontend-infra` on `irc.mozilla.com` can answer your questions.
```
Comment (helfi92) [inline src/data/firefox-health-dashboard.yaml]: `s/irc/irccloud/`.

Comment (helfi92) [reply src/data/firefox-health-dashboard.yaml]: I think a link to https://wiki.mozilla.org/IRC might be beneficial to properly set up IRC. In `taskcluster.yml` we wrote:

> We are also available on [irc](https://wiki.mozilla.org/IRC) in the `#taskcluster-contributors` channel.

File: src/data/firefox-health-dashboard.yaml (hunk 1 above)
Comment (klahnakoski) [reply src/data/firefox-health-dashboard.yaml]: yes, just fixing it now

**Comment by djmitche** [comment]
Wow, my Outreachy streams are crossing!

**Pull request merged**